### PR TITLE
New: adds description to UBL format selector

### PR DESCRIPTION
**Pull request by alexmigf**

### CODE REVIEW

File: includes/Documents/Invoice.php [hunk 1]
```diff
@@ -576,6 +576,38 @@ public function get_pdf_settings_fields( $option_name ) {
...
+				}
+			}
+		}
+		
+		$ubl_format_description .= sprintf(
+			/* translators: %1$s: opening link tag, %2$s: closing link tag */
+			__( 'If the format you need isn\'t listed, please don\'t hesitate to %1$scontact us%2$s!', 'woocommerce-pdf-invoices-packing-slips' ),
+			'<a href="https://wpovernight.com/contact/" target="_blank">',
+			'</a>'
+		);
```
Comment (MohamadNateqi) [inline includes/Documents/Invoice.php:608]: Maybe having this in a separate method?

File: includes/Documents/Invoice.php [hunk 2]
```diff
@@ -576,6 +576,38 @@ public function get_pdf_settings_fields( $option_name ) {
 	 * UBL settings fields
 	 */
 	public function get_ubl_settings_fields( $option_name ) {
+		$extensions_available   = array();
+		$ubl_format_description = '';
+		
+		if ( ! class_exists( 'WPO_IPS_XRechnung' ) ) {
+			$extensions_available['xrechnung'] = array(
+				'title' => __( 'EN16931 XRechnung', 'woocommerce-pdf-invoices-packing-slips' ),
+				'url'   => 'https://github.com/wpovernight/wpo-ips-xrechnung/releases/latest/',
```
Comment (MohamadNateqi) [inline includes/Documents/Invoice.php:585]: I was wondering if it's OK to put a link to GitHub here. Just asking.

Comment (alexmigf) [reply includes/Documents/Invoice.php]: I believe it simplifies the process of downloading the extension, reducing the need for users to contact us solely for that purpose.

File: includes/Documents/Invoice.php [hunk 3]
```diff
@@ -711,6 +712,46 @@ public function get_settings_categories( string $output_format ): array {
 
 		return apply_filters( 'wpo_wcpdf_document_settings_categories', $settings_categories[ $output_format ] ?? array(), $output_format, $this );
 	}
+	
+	/**
+	 * Get UBL Format setting derscription
```
Comment (MohamadNateqi) [inline includes/Documents/Invoice.php:717]: `derscription` typo

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Pull request merged**